### PR TITLE
Recipes list

### LINKED ISSUE #5: Recipes list [3pts]
# Recipes list [3pts]

- Should display a list of recipes created by the logged-in user as in the wireframe.
- Should lead to recipe details.
- If the user is the owner of the recipe, should allow user to delete it.

image

### PR DESCRIPTION
**Pull request by eapenzacharias**
- [x] Should display a list of recipes created by the logged-in user as in the wireframe.
- [x] Should lead to recipe details.
- [x] If the user is the owner of the recipe, should allow user to delete it.
- [x] If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@
   get 'foods/index'
   get 'inventories/index'
   get 'recipes/index'
+  resources :recipes, only: [:index, :show, :destroy]
```
Comment (piero-vic) [inline config/routes.rb]: We can now remove `get 'recipes/index'`.

**Review by piero-vic — CHANGES_REQUESTED** [review]
Hi @eapenzacharias 

## Changes Required! :recycle: 

We're almost done. The only thing you need to add is handle if there is no user logged in. Everything else is great.

File: config/routes.rb (hunk 1 above)
Comment (eapenzacharias) [reply config/routes.rb]: Hey @piero-vic 
Removed the unused routes and updated the navbar that used the link

**Review by piero-vic — APPROVED** [review]
## Approved :fire: 
We just need to solve the conflicts. 
- Keep the resources on `routes.rb`
- Keep both the recipes and the inventories on `seeds.rb`
- Keep your version on `_navbar.html.erb`

**Pull request merged**